guardian/testapp/migrations/0006_auto_20230727_0658.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.2.20 on 2023-07-27 06:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
At some point we should consider using a mix of both UUID primary keys and BigInt ( any non-char pks ) as we have experienced issues both in django proper and guardian upgrades with UUID fields.

Reply:
I'm not sure what this is about, but I understand this is for later. Leaving it without change.